.github/workflows/releaseinfo.yml

```diff
@@ -0,0 +1,26 @@
+name: write release information to file
+
+on: workflow_dispatch
+
+jobs:
+  release_info:
+    runs-on: ubuntu-latest
+    steps:
+      - id: info
+        uses: HarikrishnanBalagopal/nextrelease@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - run: echo '${{ steps.info.outputs.release_info }}' > _data/releaseinfo.json
```

Comment:
Do we need the "_"? The data in it is similar to bin, scripts and assets, rather than like _include,..

Reply:
yes _data is for data files. https://jekyllrb.com/docs/datafiles/

Reply:
Okay cool